proto/kms.proto

```diff
@@ -12,10 +12,11 @@ service KmsEndpoint {
 
 enum FheType {
   Bool = 0;
```

Comment:
Should we call it Ebool?

Reply:
I didn't change that, it was already `Bool` in the kms